### DISCUSSION TITLE
Send command to pause the cleaning

### DISCUSSION THREAD

**Discussion by llucax** [Tasshack/dreame-vacuum#394] [Q&A] [answered]
Hi, I'm trying to write an automation where I would like to send a command to the robot to pause/resume the cleaning.

I guess I need to use [`vacuum.send_command`](https://github.com/Tasshack/dreame-vacuum/blob/master/docs/services.md#vacuumsend_command) but in the link provided there I couldn't find any list of available commands and parameters.

Can anyone provide a direct link to the list of available commands and their parameters?

Thanks!

**Comment by Tasshack** [comment, marked as answer]
You should never use `send_command` unless you really know what your doing.

https://www.home-assistant.io/integrations/vacuum/#service-vacuumpause

**Reply by llucax** [reply]
Thanks!
